<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TracerEditWidget</class>
 <widget class="QWidget" name="TracerEditWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>394</width>
    <height>418</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,1,0">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Target:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="target">
       <property name="styleSheet">
        <string notr="true">font-weight: bold;</string>
       </property>
       <property name="text">
        <string>&lt;target&gt;</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="prefixesTable"/>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Output</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QCheckBox" name="logToStdout">
        <property name="toolTip">
         <string>Log Tracer output from this application to the process's Standard Output.</string>
        </property>
        <property name="text">
         <string>stdout</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="logToSafirLogging">
        <property name="toolTip">
         <string>Log Tracer output from this application
to Safir Logging as Debug level logs.</string>
        </property>
        <property name="text">
         <string>Safir Logging</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="logToTracer">
        <property name="toolTip">
         <string>Log Tracer output from this application to the Tracer UDP
protocol, meaning it will show up in this application.</string>
        </property>
        <property name="text">
         <string>Tracer UDP</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
</ui>
